src/nls/root/strings.js

```diff
@@ -398,6 +398,7 @@ define({
     "CMD_WORKINGSET_SORT_BY_TYPE"         : "Sort by Type",
     "CMD_WORKING_SORT_TOGGLE_AUTO"        : "Automatic Sort",
     "CMD_THEMES"                          : "Themes\u2026",
+    "CMD_TOGGLE_SEARCH_AUTOHIDE"          : "Automatically close search",
```

Comment:
I opted to change the description, since the action appears as a 'close' to the user (the search has an 'x' on it)